.jenkins/infrastructure/e2e_testing.Jenkinsfile

```diff
@@ -100,8 +103,11 @@ pipeline {
                     string(name: 'UBUNTU_2004_ICX_CUSTOM_LABEL', value: params.UBUNTU_2004_ICX_LABEL),
                     string(name: 'UBUNTU_2004_NONSGX_LABEL', value: params.UBUNTU_NONSGX_LABEL),
                     string(name: 'WS2019_DCAP_CFL_LABEL', value: params.WINDOWS_2019_DCAP_LABEL),
-                    string(name: 'WS2019_DCAP_ICX_LABEL', value: params.WINDOWS_2019_DCAP_LABEL),
+                    string(name: 'WS2019_DCAP_ICX_LABEL', value: params.WINDOWS_2019_DCAP_ICX_LABEL),
                     string(name: 'WINDOWS_NONSGX_CUSTOM_LABEL', value: params.WINDOWS_NONSGX_CUSTOM_LABEL),
+                    string(name: 'WS2022_DCAP_CFL_LABEL', value: params.WINDOWS_2022_DCAP_LABEL),
+                    string(name: 'WS2022_DCAP_ICX_LABEL', value: params.WINDOWS_2022_DCAP_ICX_LABEL),
+                    string(name: 'WINDOWS_2022_NONSGX_CUSTOM_LABEL', value: params.WINDOWS_2022_NONSGX_CUSTOM_LABEL),
```

Comment:
Please make this change in Jenkins CasC too

Reply:
Added